### ISSUE TITLE
Precompiling error

### ISSUE THREAD

**Issue by Mo8it** [JuliaPlots/Makie.jl#1511]
I get an error after running `Pkg.update()`. The updated packages are:
- Tables
- Compat
- Static
- Interpolations

Makie is dependent on all of them.

---

# Error

julia> using CairoMakie
[ Info: Precompiling CairoMakie [13f3f980-e62b-5c42-98c6-ff1f3baf88f0]
WARNING: Method definition quantile(AbstractArray{var"#s261", N} where N where var"#s261"<:Real) in module StatsBase at deprecated.jl:70 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition stdm(AbstractArray{T, N} where N where T<:Real, AbstractArray{T, N} where N where T<:Real, Int64) in module StatsBase at /home/mo/.julia/packages/StatsBase/CuCbN/src/moments.jl:141 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition stdm##kw(Any, typeof(Statistics.stdm), AbstractArray{T, N} where N where T<:Real, AbstractArray{T, N} where N where T<:Real, Int64) in module StatsBase at /home/mo/.julia/packages/StatsBase/CuCbN/src/moments.jl:141 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition describe(Any) in module StatsBase at /home/mo/.julia/packages/StatsBase/CuCbN/src/scalarstats.jl:716 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition describe(IO, AbstractArray{T, N} where N where T) in module StatsBase at /home/mo/.julia/packages/StatsBase/CuCbN/src/scalarstats.jl:721 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition describe(IO, AbstractArray{T, N} where N) where {T<:Union{Base.Missing, Real}} in module StatsBase at /home/mo/.julia/packages/StatsBase/CuCbN/src/scalarstats.jl:717 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition pairwise(typeof(Statistics.cor), Any) in module StatsBase at /home/mo/.julia/packages/StatsBase/CuCbN/src/pairwise.jl:312 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition pairwise(typeof(Statistics.cov), Any) in module StatsBase at /home/mo/.julia/packages/StatsBase/CuCbN/src/pairwise.jl:305 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition pairwise(typeof(Statistics.cov), Any, Any) in module StatsBase at /home/mo/.julia/packages/StatsBase/CuCbN/src/pairwise.jl:298 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition pairwise(Any, Any, Any) in module StatsBase at /home/mo/.julia/packages/StatsBase/CuCbN/src/pairwise.jl:282 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition pairwise(Any, Any) in module StatsBase at /home/mo/.julia/packages/StatsBase/CuCbN/src/pairwise.jl:282 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition pairwise!##kw(Any, typeof(StatsAPI.pairwise!), typeof(Statistics.cor), AbstractArray{T, 2} where T, Any) in module StatsBase at /home/mo/.julia/packages/StatsBase/CuCbN/src/pairwise.jl:309 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition pairwise!##kw(Any, typeof(StatsAPI.pairwise!), typeof(Statistics.cov), AbstractArray{T, 2} where T, Any) in module StatsBase at /home/mo/.julia/packages/StatsBase/CuCbN/src/pairwise.jl:302 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition pairwise!##kw(Any, typeof(StatsAPI.pairwise!), typeof(Statistics.cov), AbstractArray{T, 2} where T, Any, Any) in module StatsBase at /home/mo/.julia/packages/StatsBase/CuCbN/src/pairwise.jl:295 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition pairwise!##kw(Any, typeof(StatsAPI.pairwise!), Any, AbstractArray{T, 2} where T, Any, Any) in module StatsBase at /home/mo/.julia/packages/StatsBase/CuCbN/src/pairwise.jl:219 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition pairwise!##kw(Any, typeof(StatsAPI.pairwise!), Any, AbstractArray{T, 2} where T, Any) in module StatsBase at /home/mo/.julia/packages/StatsBase/CuCbN/src/pairwise.jl:219 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition pairwise##kw(Any, typeof(StatsAPI.pairwise), typeof(Statistics.cor), Any) in module StatsBase at /home/mo/.julia/packages/StatsBase/CuCbN/src/pairwise.jl:312 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition pairwise##kw(Any, typeof(StatsAPI.pairwise), typeof(Statistics.cov), Any) in module StatsBase at /home/mo/.julia/packages/StatsBase/CuCbN/src/pairwise.jl:305 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition pairwise##kw(Any, typeof(StatsAPI.pairwise), typeof(Statistics.cov), Any, Any) in module StatsBase at /home/mo/.julia/packages/StatsBase/CuCbN/src/pairwise.jl:298 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition pairwise##kw(Any, typeof(StatsAPI.pairwise), Any, Any, Any) in module StatsBase at /home/mo/.julia/packages/StatsBase/CuCbN/src/pairwise.jl:282 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition pairwise##kw(Any, typeof(StatsAPI.pairwise), Any, Any) in module StatsBase at /home/mo/.julia/packages/StatsBase/CuCbN/src/pairwise.jl:282 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition pairwise!(typeof(Statistics.cor), AbstractArray{T, 2} where T, Any) in module StatsBase at /home/mo/.julia/packages/StatsBase/CuCbN/src/pairwise.jl:309 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition pairwise!(typeof(Statistics.cov), AbstractArray{T, 2} where T, Any) in module StatsBase at /home/mo/.julia/packages/StatsBase/CuCbN/src/pairwise.jl:302 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition pairwise!(typeof(Statistics.cov), AbstractArray{T, 2} where T, Any, Any) in module StatsBase at /home/mo/.julia/packages/StatsBase/CuCbN/src/pairwise.jl:295 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition pairwise!(Any, AbstractArray{T, 2} where T, Any, Any) in module StatsBase at /home/mo/.julia/packages/StatsBase/CuCbN/src/pairwise.jl:219 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition pairwise!(Any, AbstractArray{T, 2} where T, Any) in module StatsBase at /home/mo/.julia/packages/StatsBase/CuCbN/src/pairwise.jl:219 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition Type##kw(Any, Type{Base.MPFR.BigFloat}, Base.Irrational{:mad_constant}) in module StatsBase at irrationals.jl:182 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition (::Type{Float64})(Base.Irrational{:mad_constant}) in module StatsBase at irrationals.jl:191 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition (::Type{Base.MPFR.BigFloat})(Base.Irrational{:mad_constant}) in module StatsBase at irrationals.jl:182 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

WARNING: Method definition (::Type{Float32})(Base.Irrational{:mad_constant}) in module StatsBase at irrationals.jl:192 overwritten in module StatsBase on the same line (check for duplicate calls to `include`).
  ** incremental compilation may be fatally broken for this module **

┌ Warning: Replacing module `StatsBase`
└ @ Base loading.jl:1036
┌ Warning: Replacing module `Distributions`
└ @ Base loading.jl:1036
ERROR: LoadError: importing StatsBase into Makie conflicts with an existing identifier
Stacktrace:
 [1] include(x::String)
   @ Makie ~/.julia/packages/Makie/gQOQF/src/Makie.jl:1
 [2] top-level scope
   @ ~/.julia/packages/Makie/gQOQF/src/Makie.jl:129
 [3] top-level scope (repeats 2 times)
   @ none:1
in expression starting at /home/mo/.julia/packages/Makie/gQOQF/src/stats/hist.jl:11
in expression starting at /home/mo/.julia/packages/Makie/gQOQF/src/Makie.jl:1
ERROR: LoadError: Failed to precompile Makie [ee78f7c6-11fb-53f2-987a-cfe4a2b5a57a] to /home/mo/.julia/compiled/v1.7/Makie/jl_qCg8Ym.
Stacktrace:
 [1] top-level scope (repeats 2 times)
   @ none:1
in expression starting at /home/mo/.julia/packages/CairoMakie/h8f56/src/CairoMakie.jl:1
ERROR: Failed to precompile CairoMakie [13f3f980-e62b-5c42-98c6-ff1f3baf88f0] to /home/mo/.julia/compiled/v1.7/CairoMakie/jl_EbJ6b1.

---

# Versioninfo

Julia Version 1.7.0
Commit 3bf9d17731 (2021-11-30 12:12 UTC)
Platform Info:
  OS: Linux (x86_64-redhat-linux)
  CPU: AMD Ryzen 7 5800H with Radeon Graphics
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-12.0.1 (ORCJIT, znver3)

---

# Manifest

pkg> status -m
  [621f4979] AbstractFFTs v1.0.1
  [1520ce14] AbstractTrees v0.3.4
  [79e6a3ab] Adapt v3.3.1
  [27a7e980] Animations v0.4.1
  [4fba245c] ArrayInterface v3.2.1
  [67c07d97] Automa v0.8.2
  [13072b0f] AxisAlgorithms v1.0.1
  [ab4f0b2a] BFloat16s v0.2.0
  [6e4b80f9] BenchmarkTools v1.2.2
  [62783981] BitTwiddlingConvenienceFunctions v0.1.1
  [fa961155] CEnum v0.4.1
  [2a0fbf3d] CPUSummary v0.1.6
  [00ebfdb7] CSTParser v3.3.0
  [052768ef] CUDA v3.5.0
  [159f3aea] Cairo v1.0.5
  [13f3f980] CairoMakie v0.6.6
  [69e1c6dd] CellListMap v0.6.7
  [082447d4] ChainRules v1.16.0
  [d360d2e6] ChainRulesCore v1.11.2
  [9e997f8a] ChangesOfVariables v0.1.2
  [9de3a189] CircularArrayBuffers v0.1.4
  [fb6a15b2] CloseOpenIntervals v0.1.4
  [944b1d66] CodecZlib v0.7.0
  [a2cac450] ColorBrewer v0.4.0
  [35d6a980] ColorSchemes v3.15.0
  [3da002f7] ColorTypes v0.11.0
  [c3611d14] ColorVectorSpace v0.9.8
  [5ae59095] Colors v0.12.8
  [a80b9123] CommonMark v0.8.4
  [d842c3ba] CommonRLInterface v0.3.1
  [bbf7d656] CommonSubexpressions v0.3.0
  [34da2185] Compat v3.41.0
  [187b0558] ConstructionBase v1.3.0
  [d38c429a] Contour v0.5.7
  [a8cc5b0e] Crayons v4.0.4
  [9a962f9c] DataAPI v1.9.0
  [864edb3b] DataStructures v0.18.11
  [e2d170a0] DataValueInterfaces v1.0.0
  [b429d917] DensityInterface v0.4.0
  [163ba53b] DiffResults v1.0.3
  [b552c78f] DiffRules v1.9.0
  [31c24e10] Distributions v0.25.36
  [ffbed154] DocStringExtensions v0.8.6
  [fdbdab4c] ElasticArrays v1.2.9
  [da5c29d0] EllipsisNotation v1.1.3
  [e2ba6199] ExprTools v0.1.6
  [c87230d0] FFMPEG v0.4.1
  [7a1cc6ca] FFTW v1.4.5
  [5789e2e9] FileIO v1.11.2
  [1a297f60] FillArrays v0.12.7
  [53c48c17] FixedPointNumbers v0.8.4
  [08572546] FlameGraphs v0.2.7
  [587475ba] Flux v0.12.8
  [59287772] Formatting v0.4.2
  [f6369f11] ForwardDiff v0.10.24
  [b38be410] FreeType v4.0.0
  [663a7486] FreeTypeAbstraction v0.9.5
  [d9f16b24] Functors v0.2.7
  [f7f18e0c] GLFW v3.4.1
  [e9467ef8] GLMakie v0.4.7
  [0c68f7d7] GPUArrays v8.1.2
  [61eb1bfa] GPUCompiler v0.13.10
  [5c1252a2] GeometryBasics v0.4.1
  [a2bd30eb] Graphics v1.1.1
  [3955a311] GridLayoutBase v0.6.4
  [42e2da0e] Grisu v1.0.2
  [4c0ca9eb] Gtk v1.1.10
  [27996c0f] GtkReactive v1.0.6
  [3e5b6fbb] HostCPUFeatures v0.1.5
  [0e44f5e4] Hwloc v2.0.0
  [7869d1d1] IRTools v0.4.4
  [615f187c] IfElse v0.1.1
  [a09fc81d] ImageCore v0.9.3
  [82e4d734] ImageIO v0.5.9
  [9b13fd28] IndirectArrays v1.0.0
  [d25df0c9] Inflate v0.1.2
  [842dd82b] InlineStrings v1.1.1
  [a98d9a8b] Interpolations v0.13.5
  [8197267c] IntervalSets v0.5.3
  [d8418881] Intervals v1.5.0
  [3587e190] InverseFunctions v0.1.2
  [92d709cd] IrrationalConstants v0.1.1
  [f1662d9f] Isoband v0.1.1
  [c8e1da08] IterTools v1.4.0
  [82899510] IteratorInterfaceExtensions v1.0.0
  [033835bb] JLD2 v0.4.15
  [692b3bcd] JLLWrappers v1.3.0
  [682c06a0] JSON v0.21.2
  [0f8b85d8] JSON3 v1.9.2
  [98e50ef6] JuliaFormatter v0.20.4
  [e5e0dc1b] Juno v0.8.4
  [5ab0869b] KernelDensity v0.6.3
  [929cbde3] LLVM v4.7.0
  [b964fa9f] LaTeXStrings v1.3.0
  [10f19ff3] LayoutPointers v0.1.4
  [1d6d02ad] LeftChildRightSiblingTrees v0.1.2
  [2ab3a3ac] LogExpFunctions v0.3.5
  [bdcacae8] LoopVectorization v0.12.99
  [ae8d54c2] Luxor v2.17.0
  [1914dd2f] MacroTools v0.5.9
  [ee78f7c6] Makie v0.15.3
  [20f20a25] MakieCore v0.1.3
  [d125e4d3] ManualMemory v0.1.6
  [dbb5928d] MappedArrays v0.4.1
  [7eb4fadd] Match v1.2.0
  [0a4f8689] MathTeXEngine v0.2.1
  [e89f7d12] Media v0.5.0
  [7269a6da] MeshIO v0.4.9
  [85b6ec6f] MethodAnalysis v0.4.4
  [e1d29d7a] Missings v1.0.2
  [78c3b35d] Mocking v0.7.3
  [66fc600b] ModernGL v1.1.3
  [e94cdb99] MosaicViews v0.3.3
  [872c559c] NNlib v0.7.31
  [a00861dc] NNlibCUDA v0.1.11
  [77ba4419] NaNMath v0.3.5
  [f09324ee] Netpbm v1.0.2
  [510215fc] Observables v0.4.0
  [6fe1bfb0] OffsetArrays v1.10.8
  [52e1d378] OpenEXR v0.3.2
  [bac558e1] OrderedCollections v1.4.1
  [90014a1f] PDMats v0.11.5
  [f57f5aa1] PNGFiles v0.3.12
  [19eb6ba3] Packing v0.4.2
  [5432bcbf] PaddedViews v0.5.10
  [d96e819e] Parameters v0.12.3
  [69de0a69] Parsers v2.1.2
  [eebad327] PkgVersion v0.1.1
  [995b91a9] PlotUtils v1.1.0
  [1d0040c9] PolyesterWeave v0.1.2
  [647866c9] PolygonOps v0.1.2
  [21216c6a] Preferences v1.2.2
  [c46f51b8] ProfileView v0.6.11
  [92933f4c] ProgressMeter v1.7.1
  [1fd47b50] QuadGK v2.4.2
  [74087812] Random123 v1.4.2
  [e6cf234a] RandomNumbers v1.5.3
  [c84ed2f1] Ratios v0.4.2
  [a223df75] Reactive v0.8.3
  [c1ae055f] RealDot v0.1.0
  [3cdcf5f2] RecipesBase v1.2.1
  [189a3867] Reexport v1.2.2
  [158674fc] ReinforcementLearning v0.10.0
  [e575027e] ReinforcementLearningBase v0.9.7
  [de1b191a] ReinforcementLearningCore v0.8.7
  [25e41dd2] ReinforcementLearningEnvironments v0.6.11
  [d607f57d] ReinforcementLearningZoo v0.5.6
  [05181044] RelocatableFolders v0.1.3
  [ae029012] Requires v1.2.0
  [79098fc4] Rmath v0.7.0
  [d5f540fe] RoundingIntegers v1.0.0
  [c4c386cf] Rsvg v1.0.0
  [fdea26ae] SIMD v3.3.1
  [3cdde19b] SIMDDualNumbers v0.1.0
  [94e857df] SIMDTypes v0.1.0
  [476501e8] SLEEFPirates v0.6.28
  [7b38b023] ScanByte v0.3.0
  [6c6a2e73] Scratch v1.1.0
  [efcf1570] Setfield v0.7.1
  [65257c39] ShaderAbstractions v0.2.8
  [992d4aef] Showoff v1.0.3
  [73760f76] SignedDistanceFields v0.4.0
  [a2af1166] SortingAlgorithms v1.0.1
  [276daf66] SpecialFunctions v1.8.1
  [cae243ae] StackViews v0.1.1
  [aedffcd0] Static v0.4.1
  [90137ffa] StaticArrays v1.2.13
  [82ae8749] StatsAPI v1.1.0
  [2913bbd2] StatsBase v0.33.13
  [4c63d2b9] StatsFuns v0.9.14
  [09ab397b] StructArrays v0.6.3
  [856f2bd8] StructTypes v1.8.1
  [3783bdb8] TableTraits v1.0.1
  [bd369af6] Tables v1.6.1
  [62fd8b95] TensorCore v0.1.1
  [8290d209] ThreadingUtilities v0.4.6
  [731e570b] TiffImages v0.5.2
  [f269a46b] TimeZones v1.7.0
  [a759f4b9] TimerOutputs v0.5.13
  [0796e94c] Tokenize v0.5.21
  [3bb67fe8] TranscodingStreams v0.9.6
  [5c2747f8] URIs v1.3.0
  [3a884ed6] UnPack v1.0.2
  [1cfade01] UnicodeFun v0.4.1
  [b8865327] UnicodePlots v2.5.0
  [3d5dd08c] VectorizationBase v0.21.22
  [efce3f68] WoodburyMatrices v0.5.5
  [a5390f91] ZipFile v0.9.4
  [e88e6eb3] Zygote v0.6.32
  [700de1a5] ZygoteRules v0.2.2
  [7b86fcea] ATK_jll v2.36.1+0
  [6e34b625] Bzip2_jll v1.0.8+0
  [83423d85] Cairo_jll v1.16.1+0
  [ee1fde0b] Dbus_jll v1.12.16+3
  [5ae413db] EarCut_jll v2.2.3+0
  [2e619515] Expat_jll v2.2.10+0
  [b22a6f82] FFMPEG_jll v4.4.0+0
  [f5851436] FFTW_jll v3.3.10+0
  [a3f928ae] Fontconfig_jll v2.13.93+0
  [d7e528f0] FreeType2_jll v2.10.4+0
  [559328eb] FriBidi_jll v1.0.10+0
  [0656b61e] GLFW_jll v3.3.5+1
  [77ec8976] GTK3_jll v3.24.30+0
  [78b55507] Gettext_jll v0.21.0+0
  [7746bdde] Glib_jll v2.68.3+1
  [3b182d85] Graphite2_jll v1.3.14+0
  [2e76f6c2] HarfBuzz_jll v2.8.1+1
  [e33a78d0] Hwloc_jll v2.5.0+0
  [905a6f67] Imath_jll v3.1.2+0
  [1d5cc7b8] IntelOpenMP_jll v2018.0.3+2
  [aacddb02] JpegTurbo_jll v2.1.0+0
  [c1c5ebd0] LAME_jll v3.100.1+0
  [dad2f222] LLVMExtra_jll v0.0.13+0
  [dd4b983a] LZO_jll v2.10.1+0
  [42c93a91] Libepoxy_jll v1.5.8+0
  [e9f186c6] Libffi_jll v3.2.2+1
  [d4300ac3] Libgcrypt_jll v1.8.7+0
  [7e76a0d4] Libglvnd_jll v1.3.0+3
  [7add5ba3] Libgpg_error_jll v1.42.0+0
  [94ce4f54] Libiconv_jll v1.16.1+1
  [4b2f31a3] Libmount_jll v2.35.0+0
  [925c91fb] Librsvg_jll v2.52.4+0
  [89763e89] Libtiff_jll v4.3.0+0
  [38a345b3] Libuuid_jll v2.36.0+0
  [856f044c] MKL_jll v2021.1.1+2
  [e7412a2a] Ogg_jll v1.3.5+0
  [18a262bb] OpenEXR_jll v3.1.1+0
  [458c3c95] OpenSSL_jll v1.1.10+0
  [efe28fd5] OpenSpecFun_jll v0.5.5+0
  [91d4177d] Opus_jll v1.3.2+0
  [2f80f16e] PCRE_jll v8.44.0+0
  [36c8627f] Pango_jll v1.47.0+0
  [30392449] Pixman_jll v0.40.1+0
  [f50d1b31] Rmath_jll v0.3.0+0
  [a2964d1f] Wayland_jll v1.19.0+0
  [2381bf8a] Wayland_protocols_jll v1.23.0+0
  [02c8fc9c] XML2_jll v2.9.12+0
  [aed1982a] XSLT_jll v1.1.34+0
  [4f6342f7] Xorg_libX11_jll v1.6.9+4
  [0c0b7dd1] Xorg_libXau_jll v1.0.9+4
  [3c9796d7] Xorg_libXcomposite_jll v0.4.5+4
  [935fb764] Xorg_libXcursor_jll v1.2.0+4
  [0aeada51] Xorg_libXdamage_jll v1.1.5+4
  [a3789734] Xorg_libXdmcp_jll v1.1.3+4
  [1082639a] Xorg_libXext_jll v1.3.4+4
  [d091e8ba] Xorg_libXfixes_jll v5.0.3+4
  [a51aa0fd] Xorg_libXi_jll v1.7.10+4
  [d1454406] Xorg_libXinerama_jll v1.1.4+4
  [ec84b674] Xorg_libXrandr_jll v1.5.2+4
  [ea2f1a96] Xorg_libXrender_jll v0.9.10+4
  [b6f176f1] Xorg_libXtst_jll v1.2.3+4
  [14d82f49] Xorg_libpthread_stubs_jll v0.1.0+3
  [c7cfdc94] Xorg_libxcb_jll v1.13.0+3
  [cc61e674] Xorg_libxkbfile_jll v1.1.0+4
  [35661453] Xorg_xkbcomp_jll v1.4.2+4
  [33bec58e] Xorg_xkeyboard_config_jll v2.27.0+4
  [c5fb5394] Xorg_xtrans_jll v1.4.0+3
  [3161d3a3] Zstd_jll v1.5.0+0
  [b437f822] adwaita_icon_theme_jll v3.33.92+5
  [de012916] at_spi2_atk_jll v2.34.1+4
  [0fc3237b] at_spi2_core_jll v2.34.0+4
  [da03df04] gdk_pixbuf_jll v2.42.6+0
  [059c91fe] hicolor_icon_theme_jll v0.17.0+3
  [bf975903] iso_codes_jll v4.3.0+4
  [9a68df92] isoband_jll v0.2.2+0
  [0ac62f75] libass_jll v0.15.1+0
  [f638f0a6] libfdk_aac_jll v2.0.2+0
  [b53b4c65] libpng_jll v1.6.38+0
  [f27f6e37] libvorbis_jll v1.3.7+0
  [1270edf5] x264_jll v2021.5.5+0
  [dfaa095f] x265_jll v3.5.0+0
  [d8fb68d0] xkbcommon_jll v0.9.1+5
  [0dad84c5] ArgTools
  [56f22d72] Artifacts
  [2a0f44e3] Base64
  [ade2ca70] Dates
  [8bb1440f] DelimitedFiles
  [8ba89e20] Distributed
  [f43a241f] Downloads
  [9fa8497b] Future
  [b77e0a4c] InteractiveUtils
  [4af54fe1] LazyArtifacts
  [b27032c2] LibCURL
  [76f85450] LibGit2
  [8f399da3] Libdl
  [37e2e46d] LinearAlgebra
  [56ddb016] Logging
  [d6f4376e] Markdown
  [a63ad114] Mmap
  [ca575930] NetworkOptions
  [44cfe95a] Pkg
  [de0858da] Printf
  [9abbd945] Profile
  [3fa0cd96] REPL
  [9a3f8284] Random
  [ea8e919c] SHA
  [9e88b42a] Serialization
  [1a1011a3] SharedArrays
  [6462fe0b] Sockets
  [2f01184e] SparseArrays
  [10745b16] Statistics
  [4607b0f0] SuiteSparse
  [fa267f1f] TOML
  [a4e569a6] Tar
  [8dfed614] Test
  [cf7118a7] UUIDs
  [4ec0a83e] Unicode
  [e66e0078] CompilerSupportLibraries_jll
  [deac9b47] LibCURL_jll
  [29816b5a] LibSSH2_jll
  [c8ffd9c3] MbedTLS_jll
  [14a3606d] MozillaCACerts_jll
  [4536629a] OpenBLAS_jll
  [05823500] OpenLibm_jll
  [83775a58] Zlib_jll
  [8e850b90] libblastrampoline_jll
  [8e850ede] nghttp2_jll
  [3f19e933] p7zip_jll

**Comment by ffreyer** [comment]
There is a duplicate import in src/stats/hist.jl...

**Comment by Mo8it** [comment]
I found the problem: It is because of SpecialFunctions.jl being at version 1.8.1 in my environment instead of 2.0.0. The downgrading is caused by other packages in the environment. CUDA.jl is one of them and I did report it https://github.com/JuliaGPU/CUDA.jl/issues/1276.

So the actual bug is: Makie is not compatible with SpecialFunctions 1.8.1. Or at least a package Makie is dependent on is not compatible with it. Can it be fixed?
